Yash Dewangan/howtorun.md

```diff
@@ -0,0 +1,8 @@
+<h1 align="center">How to run</h1>
+
+1-> Clone the branch repo
+2-> add the API key and update the Weather.py
+3-> run the Following command in terminal
+        pip install -r requirements.txt
```

Comment:
You need to change the type of requirement.txt it shown error that file is missing so change the type of file and also create virtual environment for this project and make requirement.txt of that environment only.